docs/api-reference/json/conversion-reference.md

```diff
@@ -56,6 +57,61 @@ like that in the configuration object above. These classes can be layers, views,
  provided the classes have been registered.
 
 
+## Functions and using `@@function`
+
+Any JavaScript function can be passed. For example, when this configuration of functions is passed to a
+`JSONConverter`–
+
+```js
+function calculateRadius({base, exponent}) {
+  return Math.pow(base, exponenet);
+}
+
+const configuration = {
+  ...,
+  functions: Object.assign({calculateRadius})
```

Comment:
Need  to update https://github.com/visgl/deck.gl/blob/master/docs/api-reference/json/json-configuration.md